Ghidra/Features/Base/src/main/java/ghidra/app/util/bin/format/omf/OmfFileHeader.java

```diff
@@ -370,6 +371,17 @@ else if (commentClass == OmfCommentRecord.COMMENT_CLASS_LIBMOD) {
 				segheader2.addIteratedData(iterheader);
 				lastDataBlock = iterheader;
 				break;
+			case CEXTDEF:
```

Comment:
Obviously you didn't design this part, but if you willing, I think a better approach would be to just key off the class type of `record` so we don't have to maintain 2 independent mappings of `record type -> record object`.  Ideally we could use [JEP 406: Pattern Matching for switch](https://openjdk.org/jeps/406) but it's only a Preview feature in JDK 17.  You could use if/else's though with [JEP 394: Pattern Matching for instanceof](https://openjdk.org/jeps/394).  Switch isn't as useful here anyway since there shouldn't be any cases that can take advantage of the fall-through.